app/src/main/java/org/fossasia/openevent/dbutils/dataDownloadStart.java

```diff
@@ -0,0 +1,11 @@
+package org.fossasia.openevent.dbutils;
+
+import org.fossasia.openevent.api.APIClient;
+
+/**
+ * Created by MananWason on 26-05-2015.
+ */
+public class dataDownloadStart {
```

Comment:
Class Names should begin with a capital letter. And `dataDownloadStart` has what purpose?